lib/all.js

```diff
@@ -1,13 +1,12 @@
 module.exports = function (router) {
-  router.get('/all.json', async function (req, res) {
```

Comment:
Why is this being removed?  I think this was supposed to be copied instead of moved.

Reply:
Oh Sorry, Yes you are right. Just give me few minuts. I add it